src/wallet/wallet.cpp

```diff
@@ -3135,6 +3130,11 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
         }
     }
 
+    if (nFeeRet > this->m_default_max_tx_fee) {
+        strFailReason = TransactionErrorString(TransactionError::MAX_FEE_EXCEEDED);
```

Comment:
style-nit: Maybe for the purpose of a backport, this could be inlined as a string literal?

Reply:
Don't you prefer an unclean-but-easy-cherry-pick backport?